src/main/kotlin/org/rust/lang/core/resolve/ref/RsLitExprReferenceContributor.kt

```diff
@@ -190,3 +206,32 @@ private class RsLiteralWebReferenceProvider : PsiReferenceProvider() {
         private val SPACE_SPLITTER: Regex = Regex("\\s")
     }
 }
+
+/**
+ * Current support:
+ *
+ * [rust-lang](https://github.com/rust-lang/rust)
+ * - [RsInnerAttr] / [RsOuterAttr] - `unstable` - `issue`
+ * - [RsInnerAttr]? / [RsOuterAttr] - `rustc_const_unstable` - `issue`
+ */
+private class RsLiteralGitHubIssueProvider : RsLiteralWebReferenceProviderBase() {
+
+    override fun getReferencesByStringLiteral(expr: RsLitExpr, literal: RsLiteralKind.String): Array<PsiReference> {
+        if (expr.containingCrate.origin != PackageOrigin.STDLIB) return PsiReference.EMPTY_ARRAY
+
+        val exprMetaItem = expr.parent as? RsMetaItem ?: return PsiReference.EMPTY_ARRAY
+
+        if (exprMetaItem.name != "issue") return PsiReference.EMPTY_ARRAY
+
+        val issueNumber = literal.value?.toLongOrNull() ?: return PsiReference.EMPTY_ARRAY
+        val attr = exprMetaItem.parentOfType<RsAttr>() ?: return PsiReference.EMPTY_ARRAY
+
+        return if (attr.metaItem.name in RUST_ISSUE_ATTR_NAMES) {
+            arrayOf(WebReference(expr, literal.offsets.value, "https://github.com/rust-lang/rust/issues/${issueNumber}"))
+        } else PsiReference.EMPTY_ARRAY
```

Comment:
```suggestion
        } else {
            PsiReference.EMPTY_ARRAY
        }
```

nit: Personally don't like inconsistency when one `if` branch has braces and another doesn't